client/src/frontend/components/project-collection/Constants.js

```diff
@@ -0,0 +1,9 @@
+const SQUARE = "square_inset";
+const WIDE = "wide_inset";
+const FULL = "full_bleed";
+
+export default {
+  SQUARE,
+  WIDE,
+  FULL
+};
```

Comment:
I also use it in the file that generates a mocked project collection over in [client/src/test/fixtures/entities/project-collection.js](https://github.com/ManifoldScholar/manifold/blob/cc/project-collection-headers/client/src/test/fixtures/entities/project-collection.js#L21). The spec I was basing my work off said the project collection has an enum value for the hero type that defaults to the square layout. I didn't want to repeat the string that denotes the square value in more than one file in case the value needs to be changed in the future, but would be happy to hard code it if that is considered overkill.